-------------------- Log Start | Mon Jan 20 00:18:27 2025 --------------------
platform: Linux-6.1.0-29-amd64-x86_64-with-glibc2.36
data_path: /home/secretjameser/Ender3Pro_data
is_default_data_path: False
config_file: /home/secretjameser/Ender3Pro_data/config/moonraker.conf
backup_config: /home/secretjameser/Ender3Pro_data/config/.moonraker.conf.bkp
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: f5a595eb585d48de9e462abc9c33a19b
unix_socket_path: /home/secretjameser/Ender3Pro_data/comms/moonraker.sock
software_version: v0.9.3-1-g4e00a07
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/secretjameser/Ender3Pro_data/logs/moonraker.log
python_version: 3.11.2 (main, Nov 30 2024, 21:22:50) [GCC 12.2.0]
launch_args: /home/secretjameser/moonraker-env/bin/python /home/secretjameser/moonraker/moonraker/moonraker.py -d /home/secretjameser/Ender3Pro_data
msgspec_enabled: False
uvloop_enabled: False

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /home/secretjameser/Ender3Pro_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager fluidd-config]
type = git_repo
primary_branch = master
path = ~/fluidd-config
origin = https://github.com/fluidd-core/fluidd-config.git
managed_services = klipper

[update_manager fluidd]
type = web
channel = stable
repo = fluidd-core/fluidd
path = ~/fluidd

#################################################################
All Configuration Files:
/home/secretjameser/Ender3Pro_data/config/moonraker.conf
#################################################################
Loading Sqlite database provider. Sqlite Version: 3.40.1
Unsafe Shutdown Count: 0

System Info:

***python***
  version: (3, 11, 2, 'final', 0)
  version_string: 3.11.2 (main, Nov 30 2024, 21:22:50) [GCC 12.2.0]

***cpu_info***
  cpu_count: 4
  bits: 64bit
  processor: x86_64
  cpu_desc: Intel(R) Core(TM) i5 CPU       M 460  @ 2.53GHz
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 5859492
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.1.0-29-amd64

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
Module libcamera unavailble, import failed

Systemd unit moonraker-Ender3Pro.service:
unit_name: moonraker-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['system.slice', 'network-online.target', '-.mount', 'sysinit.target']
**After=['-.mount', 'system.slice', 'basic.target', 'sysinit.target', 'systemd-journald.socket', 'network-online.target']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/moonraker.env
**ExecStart=/home/secretjameser/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/secretjameser/moonraker
**FragmentPath=/etc/systemd/system/moonraker-Ender3Pro.service
**Description=API Server for Klipper SV1 Ender3Pro
**User=secretjameser

Systemd unit klipper-Ender3Pro.service:
unit_name: klipper-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['sysinit.target', '-.mount', 'system.slice']
**After=['network-online.target', 'system.slice', 'systemd-journald.socket', 'basic.target', '-.mount', 'sysinit.target']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/klipper.env
**ExecStart=/home/secretjameser/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/secretjameser/klipper
**FragmentPath=/etc/systemd/system/klipper-Ender3Pro.service
**Description=Klipper 3D Printer Firmware SV1
**User=secretjameser
Git Repo moonraker: No warnings detected
Git Repo klipper: No warnings detected
Git Repo fluidd-config: No warnings detected
Klipper Version: v0.12.0-377-g9bd0d4757
2025-01-19 19:18:27,090 [proc_stats.py:_watchdog_callback()] - EVENT LOOP BLOCKED: 6.39 seconds, total blocked count: 4
2025-01-19 19:18:27,292 [proc_stats.py:log_last_stats()] - 
System Time: 1737172684.559040, Usage: 0.46%, Memory: 53908 kB
System Time: 1737172685.561360, Usage: 0.69%, Memory: 53908 kB
System Time: 1737172686.562207, Usage: 0.66%, Memory: 53908 kB
System Time: 1737172687.563700, Usage: 0.88%, Memory: 53908 kB
System Time: 1737332307.112516, Usage: 0.07%, Memory: 53908 kB
2025-01-19 19:19:33,405 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.38
2025-01-19 19:19:33,523 [application.py:log_request()] - 101 GET /websocket?token=JSRTXUKWWWNMMIHJL6RCSDYP3URD5QFE (192.168.1.38) [_TRUSTED_USER_] 2.06ms
2025-01-19 19:19:33,523 [websockets.py:open()] - Websocket Opened: ID: 139791149807760, Proxied: False, User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0, Host Name: 192.168.1.38
2025-01-19 19:19:33,541 [websockets.py:_handle_identify()] - Websocket 139791149807760 Client Identified - Name: fluidd, Version: 1.30.6-894aa23, Type: web
2025-01-19 19:19:33,546 [file_manager.py:get_file_list()] - Updating File List <config>...
2025-01-19 19:19:39,277 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/secretjameser/moonraker/moonraker/components/klippy_connection.py", line 802, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.11/asyncio/streams.py", line 350, in wait_closed
    await self._protocol._get_close_waiter(self)
  File "/usr/lib/python3.11/asyncio/selector_events.py", line 995, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ConnectionResetError: [Errno 104] Connection reset by peer
2025-01-19 19:19:39,278 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-01-19 19:19:39,278 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Disconnected
2025-01-19 19:19:39,278 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.objects.list, Code: 503, Message: Klippy Disconnected
2025-01-19 19:19:39,515 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Host not connected
2025-01-19 19:19:39,530 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-01-19 19:19:39,550 [machine.py:extract_service_info()] - Detected alternate unit name for klipper: klipper-Ender3Pro.service
2025-01-19 19:20:17,933 [websockets.py:on_close()] - Websocket Closed: ID: 139791149807760 Close Code: None, Close Reason: None, Pong Time Elapsed: 4.41
2025-01-19 19:21:09,661 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-01-19 19:21:09,662 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-01-19 19:21:09,663 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-01-19 19:37:35,457 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-01-19 19:37:35,460 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-01-19 19:37:38,974 [server.py:main()] - Server Shutdown
-------------------- Log Start | Mon Jan 20 00:39:18 2025 --------------------
platform: Linux-6.1.0-30-amd64-x86_64-with-glibc2.36
data_path: /home/secretjameser/Ender3Pro_data
is_default_data_path: False
config_file: /home/secretjameser/Ender3Pro_data/config/moonraker.conf
backup_config: /home/secretjameser/Ender3Pro_data/config/.moonraker.conf.bkp
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: f5a595eb585d48de9e462abc9c33a19b
unix_socket_path: /home/secretjameser/Ender3Pro_data/comms/moonraker.sock
software_version: v0.9.3-16-g72ed175
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/secretjameser/Ender3Pro_data/logs/moonraker.log
python_version: 3.11.2 (main, Nov 30 2024, 21:22:50) [GCC 12.2.0]
launch_args: /home/secretjameser/moonraker-env/bin/python /home/secretjameser/moonraker/moonraker/moonraker.py -d /home/secretjameser/Ender3Pro_data
msgspec_enabled: False
uvloop_enabled: False
2025-01-19 19:39:18,882 [confighelper.py:read_file()] - Configuration File '/home/secretjameser/Ender3Pro_data/config/moonraker.conf' parsed, total size: 725 B
2025-01-19 19:39:18,882 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /home/secretjameser/Ender3Pro_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager fluidd-config]
type = git_repo
primary_branch = master
path = ~/fluidd-config
origin = https://github.com/fluidd-core/fluidd-config.git
managed_services = klipper

[update_manager fluidd]
type = web
channel = stable
repo = fluidd-core/fluidd
path = ~/fluidd

#################################################################
All Configuration Files:
/home/secretjameser/Ender3Pro_data/config/moonraker.conf
#################################################################
2025-01-19 19:39:20,358 [server.py:load_component()] - Component (secrets) loaded
2025-01-19 19:39:20,629 [server.py:load_component()] - Component (template) loaded
2025-01-19 19:39:20,633 [server.py:load_component()] - Component (klippy_connection) loaded
2025-01-19 19:39:32,947 [server.py:load_component()] - Component (application) loaded
2025-01-19 19:39:33,226 [server.py:load_component()] - Component (websockets) loaded
2025-01-19 19:39:33,533 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-01-19 19:39:33,534 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 0
2025-01-19 19:39:33,537 [server.py:load_component()] - Component (database) loaded
2025-01-19 19:39:35,685 [server.py:load_component()] - Component (dbus_manager) loaded
2025-01-19 19:39:36,018 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-01-19 19:39:36,367 [server.py:load_component()] - Component (file_manager) loaded
2025-01-19 19:39:36,942 [database.py:register_table()] - Found registered table authorized_users
2025-01-19 19:39:36,944 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
10.0.0.0/8
127.0.0.0/8
169.254.0.0/16
172.16.0.0/12
192.168.0.0/16
fe80::/10
::1/128
CORS Domains:
.*\.lan
.*\.local
.*://localhost
.*://localhost:.*
.*://my\.mainsail\.xyz
.*://app\.fluidd\.xyz
2025-01-19 19:39:36,946 [server.py:load_component()] - Component (authorization) loaded
2025-01-19 19:39:36,987 [server.py:load_component()] - Component (klippy_apis) loaded
2025-01-19 19:39:38,206 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 11, 2, 'final', 0)
  version_string: 3.11.2 (main, Nov 30 2024, 21:22:50) [GCC 12.2.0]

***cpu_info***
  cpu_count: 4
  bits: 64bit
  processor: x86_64
  cpu_desc: Intel(R) Core(TM) i5 CPU       M 460  @ 2.53GHz
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 5859492
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.1.0-30-amd64

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-01-19 19:39:38,268 [server.py:load_component()] - Component (shell_command) loaded
2025-01-19 19:39:38,268 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-01-19 19:39:38,304 [server.py:add_log_rollover_item()] - Module libcamera unavailble, import failed
2025-01-19 19:39:38,304 [server.py:load_component()] - Component (machine) loaded
2025-01-19 19:39:38,394 [server.py:load_component()] - Component (data_store) loaded
2025-01-19 19:39:38,424 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-01-19 19:39:38,424 [server.py:load_component()] - Component (proc_stats) loaded
2025-01-19 19:39:38,444 [server.py:load_component()] - Component (job_state) loaded
2025-01-19 19:39:38,470 [server.py:load_component()] - Component (job_queue) loaded
2025-01-19 19:39:38,520 [database.py:register_table()] - Found registered table job_history
2025-01-19 19:39:38,520 [database.py:register_table()] - Found registered table job_totals
2025-01-19 19:39:38,521 [server.py:load_component()] - Component (history) loaded
2025-01-19 19:39:38,688 [server.py:load_component()] - Component (http_client) loaded
2025-01-19 19:39:38,725 [server.py:load_component()] - Component (announcements) loaded
2025-01-19 19:39:38,745 [server.py:load_component()] - Component (webcam) loaded
2025-01-19 19:39:38,853 [server.py:load_component()] - Component (extensions) loaded
2025-01-19 19:39:39,312 [base_deploy.py:log_info()] - Git Repo moonraker: Detected virtualenv: /home/secretjameser/moonraker-env
2025-01-19 19:39:39,315 [base_deploy.py:log_info()] - Git Repo klipper: Detected virtualenv: /home/secretjameser/klippy-env
2025-01-19 19:39:39,320 [server.py:load_component()] - Component (update_manager) loaded
2025-01-19 19:39:39,364 [server.py:load_component()] - Component (octoprint_compat) loaded
2025-01-19 19:39:39,364 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-01-19 19:39:39,835 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-01-19 19:39:39,875 [server.py:_initialize_component()] - Performing Component Post Init: [file_manager]
2025-01-19 19:39:39,876 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-01-19 19:39:39,878 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-01-19 19:39:39,878 [machine.py:validation_init()] - Installation version in database up to date
2025-01-19 19:39:40,043 [machine.py:check_virt_status()] - No Virtualization Detected
2025-01-19 19:39:40,066 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.38
2025-01-19 19:39:40,066 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://192.168.1.38
2025-01-19 19:39:40,089 [machine.py:extract_service_info()] - Detected alternate unit name for moonraker: moonraker-Ender3Pro.service
2025-01-19 19:39:40,093 [server.py:add_log_rollover_item()] - 
Systemd unit moonraker-Ender3Pro.service:
unit_name: moonraker-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['-.mount', 'sysinit.target', 'system.slice', 'network-online.target']
**After=['basic.target', 'sysinit.target', '-.mount', 'system.slice', 'network-online.target', 'systemd-journald.socket']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/moonraker.env
**ExecStart=/home/secretjameser/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/secretjameser/moonraker
**FragmentPath=/etc/systemd/system/moonraker-Ender3Pro.service
**Description=API Server for Klipper SV1 Ender3Pro
**User=secretjameser
2025-01-19 19:39:40,093 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-01-19 19:39:40,093 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-01-19 19:39:40,095 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-01-19 19:39:40,097 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-01-19 19:39:40,097 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://192.168.1.38
2025-01-19 19:39:40,101 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-01-19 19:39:40,101 [server.py:add_log_rollover_item()] - 
Systemd unit klipper-Ender3Pro.service:
unit_name: klipper-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['sysinit.target', '-.mount', 'system.slice']
**After=['network-online.target', 'system.slice', 'systemd-journald.socket', 'basic.target', '-.mount', 'sysinit.target']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/klipper.env
**ExecStart=/home/secretjameser/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/secretjameser/klipper
**FragmentPath=/etc/systemd/system/klipper-Ender3Pro.service
**Description=Klipper 3D Printer Firmware SV1
**User=secretjameser
2025-01-19 19:39:40,101 [server.py:_initialize_component()] - Performing Component Post Init: [update_manager]
2025-01-19 19:39:41,221 [base_deploy.py:log_info()] - PackageDeploy: PackageKit Provider Configured
2025-01-19 19:39:41,311 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/secretjameser/Ender3Pro_data/comms/moonraker.sock'
2025-01-19 19:39:41,312 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: debian
2025-01-19 19:39:41,312 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-01-19 19:39:41,563 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-01-19 19:39:41,591 [machine.py:extract_service_info()] - Detected alternate unit name for klipper: klipper-Ender3Pro.service
2025-01-19 19:39:41,596 [server.py:add_log_rollover_item()] - 
Systemd unit klipper-Ender3Pro.service:
unit_name: klipper-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['system.slice', 'sysinit.target', '-.mount']
**After=['-.mount', 'network-online.target', 'sysinit.target', 'systemd-journald.socket', 'basic.target', 'system.slice']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/klipper.env
**ExecStart=/home/secretjameser/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/secretjameser/klipper
**FragmentPath=/etc/systemd/system/klipper-Ender3Pro.service
**Description=Klipper 3D Printer Firmware SV1
**User=secretjameser
2025-01-19 19:39:41,848 [server.py:add_log_rollover_item()] - Klipper Version: v0.12.0-410-gcf3b0475d
2025-01-19 19:39:50,448 [base_deploy.py:log_info()] - PackageKit: Detected 0 package updates:

2025-01-19 19:39:50,453 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker status --porcelain -b) successfully finished
2025-01-19 19:39:50,457 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker branch --list --no-color) successfully finished
2025-01-19 19:39:50,462 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker remote get-url origin) successfully finished
2025-01-19 19:39:51,048 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker fetch origin --prune --progress) successfully finished
2025-01-19 19:39:51,052 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker merge-base --is-ancestor HEAD origin/master) successfully finished
2025-01-19 19:39:51,057 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker rev-parse HEAD) successfully finished
2025-01-19 19:39:51,062 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker describe --always --tags --long --dirty --abbrev=8) successfully finished
2025-01-19 19:39:51,066 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker rev-parse origin/master) successfully finished
2025-01-19 19:39:51,071 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker describe origin/master --always --tags --long --abbrev=8) successfully finished
2025-01-19 19:39:51,075 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/moonraker rev-list HEAD..72ed175c520e3b9e3054e9ad7d27157577300df1 --count) successfully finished
2025-01-19 19:39:51,075 [git_deploy.py:log_repo_info()] - Git Repo moonraker Detected:
Owner: Arksine
Repository Name: moonraker
Path: /home/secretjameser/moonraker
Remote: origin
Branch: master
Remote URL: https://github.com/Arksine/moonraker.git
Recovery URL: https://github.com/Arksine/moonraker.git
Current Commit SHA: 72ed175c520e3b9e3054e9ad7d27157577300df1
Upstream Commit SHA: 72ed175c520e3b9e3054e9ad7d27157577300df1
Current Version: v0.9.3-16-g72ed175c
Upstream Version: v0.9.3-16-g72ed175c
Rollback Commit: 71f9e677b81afcc6b99dd5002f595025c38edc7b
Rollback Branch: master
Rollback Version: v0.9.3-0-g71f9e677
Is Dirty: False
Is Detached: False
Is Shallow: False
Commits Behind Count: 0
Diverged: False
Pinned Commit: None
2025-01-19 19:39:51,075 [base_deploy.py:log_info()] - Git Repo moonraker: Channel: dev
2025-01-19 19:39:51,075 [base_deploy.py:log_info()] - Git Repo moonraker: Validity check for git repo passed
2025-01-19 19:39:51,089 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper status --porcelain -b) successfully finished
2025-01-19 19:39:51,092 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper branch --list --no-color) successfully finished
2025-01-19 19:39:51,098 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper remote get-url origin) successfully finished
2025-01-19 19:39:51,598 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper fetch origin --prune --progress) successfully finished
2025-01-19 19:39:51,602 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper merge-base --is-ancestor HEAD origin/master) successfully finished
2025-01-19 19:39:51,605 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper rev-parse HEAD) successfully finished
2025-01-19 19:39:51,665 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper describe --always --tags --long --dirty --abbrev=8) successfully finished
2025-01-19 19:39:51,670 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper rev-parse origin/master) successfully finished
2025-01-19 19:39:51,723 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper describe origin/master --always --tags --long --abbrev=8) successfully finished
2025-01-19 19:39:51,727 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/klipper rev-list HEAD..cf3b0475daa0d7154d2f986f94d8c184c7cf39c1 --count) successfully finished
2025-01-19 19:39:51,727 [git_deploy.py:log_repo_info()] - Git Repo klipper Detected:
Owner: Klipper3d
Repository Name: klipper
Path: /home/secretjameser/klipper
Remote: origin
Branch: master
Remote URL: https://github.com/Klipper3d/klipper
Recovery URL: https://github.com/Klipper3d/klipper
Current Commit SHA: cf3b0475daa0d7154d2f986f94d8c184c7cf39c1
Upstream Commit SHA: cf3b0475daa0d7154d2f986f94d8c184c7cf39c1
Current Version: v0.12.0-410-gcf3b0475
Upstream Version: v0.12.0-410-gcf3b0475
Rollback Commit: b89d552387da9734292a182c7040c418996ca350
Rollback Branch: master
Rollback Version: v0.12.0-317-gb89d5523
Is Dirty: False
Is Detached: False
Is Shallow: False
Commits Behind Count: 0
Diverged: False
Pinned Commit: None
2025-01-19 19:39:51,727 [base_deploy.py:log_info()] - Git Repo klipper: Channel: dev
2025-01-19 19:39:51,727 [base_deploy.py:log_info()] - Git Repo klipper: Validity check for git repo passed
2025-01-19 19:39:52,427 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config status --porcelain -b) successfully finished
2025-01-19 19:39:52,430 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config branch --list --no-color) successfully finished
2025-01-19 19:39:52,436 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config remote get-url origin) successfully finished
2025-01-19 19:39:52,979 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config fetch origin --prune --progress) successfully finished
2025-01-19 19:39:52,983 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config merge-base --is-ancestor HEAD origin/master) successfully finished
2025-01-19 19:39:52,985 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config rev-parse HEAD) successfully finished
2025-01-19 19:39:52,989 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config describe --always --tags --long --dirty --abbrev=8) successfully finished
2025-01-19 19:39:52,992 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config rev-parse origin/master) successfully finished
2025-01-19 19:39:52,995 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config describe origin/master --always --tags --long --abbrev=8) successfully finished
2025-01-19 19:39:52,999 [shell_command.py:_check_proc_success()] - Command (git -C /home/secretjameser/fluidd-config rev-list HEAD..807175d72e3a00cdc6b5e249444a4630e1e03a55 --count) successfully finished
2025-01-19 19:39:52,999 [git_deploy.py:log_repo_info()] - Git Repo fluidd-config Detected:
Owner: fluidd-core
Repository Name: fluidd-config
Path: /home/secretjameser/fluidd-config
Remote: origin
Branch: master
Remote URL: https://github.com/fluidd-core/fluidd-config.git
Recovery URL: https://github.com/fluidd-core/fluidd-config.git
Current Commit SHA: 807175d72e3a00cdc6b5e249444a4630e1e03a55
Upstream Commit SHA: 807175d72e3a00cdc6b5e249444a4630e1e03a55
Current Version: v1.2.1-0-g807175d7
Upstream Version: v1.2.1-0-g807175d7
Rollback Commit: ce48c5854e9ca3ca72b63d0327069fac20e94c7c
Rollback Branch: master
Rollback Version: v1.2.0-0-gce48c585
Is Dirty: False
Is Detached: False
Is Shallow: False
Commits Behind Count: 0
Diverged: False
Pinned Commit: None
2025-01-19 19:39:52,999 [base_deploy.py:log_info()] - Git Repo fluidd-config: Channel: dev
2025-01-19 19:39:52,999 [base_deploy.py:log_info()] - Git Repo fluidd-config: Validity check for git repo passed
2025-01-19 19:39:53,229 [base_deploy.py:log_info()] - Web Client fluidd: Detected
Repo: fluidd-core/fluidd
Channel: stable
Path: /home/secretjameser/fluidd
Local Version: v1.31.4
Remote Version: v1.31.4
Valid: True
Fallback Detected: False
Pre-release: False
Download Url: https://github.com/fluidd-core/fluidd/releases/download/v1.31.4/fluidd.zip
Download Size: 3857394
Content Type: application/zip
Rollback Version: v1.30.5
Rollback Repo: fluidd-core/fluidd
2025-01-19 19:40:51,284 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-01-19 19:40:51,285 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-01-19 19:40:51,285 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-01-19 20:10:29,829 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.153
2025-01-19 20:10:29,831 [application.py:log_request()] - 101 GET /websocket (192.168.1.153) [_TRUSTED_USER_] 2.65ms
2025-01-19 20:10:29,831 [websockets.py:open()] - Websocket Opened: ID: 140519401539792, Proxied: True, User Agent: Mozilla/5.0 (Android 13; Mobile; rv:134.0) Gecko/134.0 Firefox/134.0, Host Name: 192.168.1.38
2025-01-19 20:10:29,849 [websockets.py:on_close()] - Websocket Closed: ID: 140519401539792 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.02
2025-01-19 20:10:29,892 [application.py:log_request()] - 101 GET /websocket (192.168.1.153) [_TRUSTED_USER_] 0.86ms
2025-01-19 20:10:29,892 [websockets.py:open()] - Websocket Opened: ID: 140519399666256, Proxied: False, User Agent: Mozilla/5.0 (Android 13; Mobile; rv:134.0) Gecko/134.0 Firefox/134.0, Host Name: 192.168.1.38
2025-01-19 20:10:29,903 [websockets.py:on_close()] - Websocket Closed: ID: 140519399666256 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.01
2025-01-19 20:10:29,969 [application.py:log_request()] - 101 GET /websocket?token=SLNKADQ2BBN46DDRBJ7FPYSUAG4SXFQR (192.168.1.153) [_TRUSTED_USER_] 0.87ms
2025-01-19 20:10:29,969 [websockets.py:open()] - Websocket Opened: ID: 140519399617680, Proxied: True, User Agent: Mozilla/5.0 (Android 13; Mobile; rv:134.0) Gecko/134.0 Firefox/134.0, Host Name: 192.168.1.38
2025-01-19 20:10:29,981 [websockets.py:_handle_identify()] - Websocket 140519399617680 Client Identified - Name: fluidd, Version: 1.31.4-f283a37, Type: web
2025-01-19 20:10:29,981 [file_manager.py:get_file_list()] - Updating File List <config>...
2025-01-19 20:10:34,446 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/secretjameser/moonraker/moonraker/components/klippy_connection.py", line 802, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.11/asyncio/streams.py", line 350, in wait_closed
    await self._protocol._get_close_waiter(self)
  File "/usr/lib/python3.11/asyncio/selector_events.py", line 995, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ConnectionResetError: [Errno 104] Connection reset by peer
2025-01-19 20:10:34,447 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-01-19 20:10:34,448 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Disconnected
2025-01-19 20:10:34,448 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.objects.list, Code: 503, Message: Klippy Disconnected
2025-01-19 20:10:34,614 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Host not connected
2025-01-19 20:10:34,700 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-01-19 20:10:34,723 [machine.py:extract_service_info()] - Detected alternate unit name for klipper: klipper-Ender3Pro.service
2025-01-19 20:12:04,780 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-01-19 20:12:04,781 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-01-19 20:12:04,782 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-01-19 20:12:27,698 [websockets.py:on_close()] - Websocket Closed: ID: 140519399617680 Close Code: None, Close Reason: None, Pong Time Elapsed: 7.53
2025-01-19 20:14:48,385 [application.py:log_request()] - 101 GET /websocket?token=ULHBRC7BLFPCXRDE4R73HULGE2QOUYMR (192.168.1.153) [_TRUSTED_USER_] 0.86ms
2025-01-19 20:14:48,385 [websockets.py:open()] - Websocket Opened: ID: 140519400350160, Proxied: False, User Agent: Mozilla/5.0 (Android 13; Mobile; rv:134.0) Gecko/134.0 Firefox/134.0, Host Name: 192.168.1.38
2025-01-19 20:14:48,471 [websockets.py:_handle_identify()] - Websocket 140519400350160 Client Identified - Name: fluidd, Version: 1.31.4-f283a37, Type: web
2025-01-19 20:14:48,471 [file_manager.py:get_file_list()] - Updating File List <config>...
2025-01-19 20:14:50,506 [websockets.py:on_close()] - Websocket Closed: ID: 140519400350160 Close Code: None, Close Reason: None, Pong Time Elapsed: 2.12
2025-01-19 21:19:39,902 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.1.153
